raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -654,4 +665,56 @@ describe('Raiden', () => {
       }),
     ).resolves.toEqual(channelOpenHash);
   });
+
+  test('closeChannel', async () => {
+    const closeChannelHash = makeHash();
+    const closeBlock = 20;
+    const channelOpenedState: RaidenState = channelOpenSuccessReducer(
```

Comment:
Same here: if you just want to fill the `channels` field, why not do it directly?

Reply:
Yes, @agatsoh , I agree with Paul. You may need to use the open reducer and epic in case you're testing channel opening. After that, you should use/implement the bare minimum of the logic needed to test your specific test, and assign/mock directly anything that was already tested before and is *known* to work

Reply:
I am just re-using the code that I used in `openChannel` and its going to be re-used in `depositChannel` as well, it makes the our job simpler

Reply:
Its the same  what you would do to fill `channels` field just that `channelOpenSuccessReducer` makes it all the more easier to fill this channels field and we re-use code across the three tests.

Reply:
@palango @andrevmatos I would anyhow without using the `channelOpenSuccessReducer` need to fill up the `Channel` type, so thats why I use the `channelOpenSuccessReducer` as a utility or helper function to help me fill it out. But if you still want me to not use it then I will make necessary changes. Waiting for your thoughts again on this.

Reply:
@agatsoh you can create a utility to return you a filled state, or to create a Channel state for you which you can assign to your state. I know this can look as some cluttering, but this is actually how unit testing is supposed to work, or else you'd end up reimplementing the whole set of reducers and using it in place.

Reply:
@andrevmatos ok i have made the necessary changes so you could give another look.